query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/aggregation/group_by_having.rs

```diff
@@ -54,12 +54,9 @@ mod aggregation_group_by_having {
                 _sum { int }
               }
             }"#,
-            vec![
-                r#"{"data":{"groupByTestModel":[{"string":"group1","int":5,"_count":{"_all":1},"_sum":{"int":5}},{"string":"group2","int":5,"_count":{"_all":1},"_sum":{"int":5}}]}}"#, // SQL
-                r#"{"data":{"groupByTestModel":[{"string":"group2","int":5,"_count":{"_all":1},"_sum":{"int":5}},{"string":"group1","int":5,"_count":{"_all":1},"_sum":{"int":5}}]}}"# // Mongo
-            ]
+            [SqlServer, Postgres, MySql, Sqlite, MongoDb] => r#"{"data":{"groupByTestModel":[{"string":"group1","int":5,"_count":{"_all":1},"_sum":{"int":5}},{"string":"group2","int":5,"_count":{"_all":1},"_sum":{"int":5}}]}}"#,
```

Comment:
MongoDB is in both branches?

Reply:
Right. MongoDB 4.2 and Mongo 5 give different results.